src/applications/apps.js

```diff
@@ -29,7 +29,7 @@ export function declareChildApplication(appName, arg1, arg2) {
 export function registerApplication(appName, arg1, arg2, customProps = {}) {
   if (typeof appName !== 'string' || appName.length === 0)
     throw new Error(`The first argument must be a non-empty string 'appName'`);
-  if (apps[appName])
+  if (getAppNames().indexOf(appName) !== -1)
```

Comment:
The actual fix for #165. I'm using `indexOf` because `contains` isn't supported by IE